rlberry/agents/bandits/tools/tracker.py

```diff
@@ -60,13 +106,32 @@ def reset_tracker(self):
                 tag_scalar_dict["iw_total_reward"] = 0.0
             self.add_scalars(arm, tag_scalar_dict)
 
+    @property
+    def t(self):
+        return self.read_last_tag_value("t")
+
+    def n_pulls(self, arm):
+        return self.read_last_tag_value("n_pulls", arm)
+
+    def rewards(self, arm):
+        return self.read_tag_value("reward", arm)
+
+    def total_reward(self, arm):
+        return self.read_last_tag_value("total_reward", arm)
+
+    def mu_hat(self, arm):
+        return self.read_last_tag_value("mu_hat", arm)
+
+    def iw_total_reward(self, arm):
+        return self.read_last_tag_value("iw_total_reward", arm)
+
     def update(self, arm, reward, params={}):
-        self.add_scalar("t", self.read_last_tag_value("t") + 1)
```

Comment:
Maybe a one-line docstring for each of those methods would be good ? I mean, it is mostly obvious from the name of the reward but a little docstring would not hurt. In particular to explain what iw_total_reward is and what does the update function do.